resources/ec2_security_groups.go

```diff
@@ -475,3 +481,9 @@ func fetchEc2SecurityGroupIpPermissionsEgressUserIdGroupPairs(ctx context.Contex
 	res <- securityGroupIpPermissionEgress.UserIdGroupPairs
 	return nil
 }
+
+func resolveSGArn(_ context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	cl := meta.(*client.Client)
+	sg := resource.Item.(types.SecurityGroup)
+	return resource.Set(c.Name, client.GenerateResourceARN("ec2", "security-group", *sg.GroupId, cl.Region, cl.AccountID))
```

Comment:
I think `if sg != nil` check is needed here.

Reply:
no need since we just casted it